src/main/steps/tasklistresponse/summary/content.ts

```diff
@@ -178,9 +178,10 @@ export const enInternationalContent = {
     title: '6. International element',
   },
   keys: {
-    start: 'Do the children live outside of England or Wales?',
```

Comment:
Can't the contents from the IE or respective modules be exported and used here, so it does not have to be updated in multiple places?

Reply:
Changed to use content from the IE check your answers page